samples/tfm/tfm_hello_world/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.13.1)
```

Comment:
@oyvindronningstad doesn't tfm now require cmake > 3.15.0?
does this need to be reflected here? 
@tejlmand ^^

Reply:
I thought about it. @magnev has also brought this up. I thought maybe that would be covered in the TF-M Cmake files.

Reply:
yes, it's also covered in TFM CMake file, but when the sample itself is a tfm based sample, then we should inform the user directly, instead of waiting for the TFM cmake part to fail.

Will also work as information to anyone looking directly at the sample/CMakeLists.txt file, that this sample itself requires 3.15.

Things are a bit trickier for samples that can be used both with and without TFM, cause in those cases, 3.15 is the minimum seen from the sample.

Reply:
I can update it no problem, but I don't want users who enable TFM to have to remember to do this.

I'll update it and add a comment.

Reply:
Done